www/layouts/_default/baseof.html

```diff
@@ -20,10 +20,13 @@
   <header>
     {{ partial "header.html" . }}
   </header>
+  </div>
   {{ partial "menu.html" (dict "menuID" "main" "page" .) }}
   <main>
-    <h1>{{- block "title" . -}}{{ .Title }}{{- end -}}</h1>
-    {{ block "main" . }}{{ end }}
+    <div class="inner-container">
```

Comment:
This `.inner-container` class seems unused. I know older webdevs and young punks like to put `<div>s` everywhere “just in case”, but I’m clearly on the other side of the fence here. ^^